packages/frontend/components/Sidebar/NUPathSection.tsx

```diff
@@ -49,6 +49,14 @@ const NUPathSection: React.FC<NUPathSectionProps> = ({
   const nupathMap: Record<string, number> = {};
 
   for (const course of coursesTaken) {
+    // Check if the course is a COOP, if it is, check off the Integration Experience
+    if (
+      course.name === COOP_BLOCK.name &&
```

Comment:
this is a good idea and it makes sense since COOP_BLOCK is a special case, but is there any way to update the structure of the COOP_BLOCK object itself to make it work?